### ISSUE TITLE
CUD for images

### ISSUE THREAD

**Issue by TLemur** [totravel/minidocx#5]

**Comment by 7Towers** [comment]
Is there a way to insert images into a document, currently?

**Comment by totravel** [comment]
Nope. This project needs to be refactored to support images insertion, but I don't have time to do it.

**Comment by 7Towers** [comment]
Understood, no problem. Is it a major refactor? Something I could contribute with some initial direction?
